ui/src/app/services/conversion/conversion.service.ts

```diff
@@ -198,16 +221,22 @@ export class ConversionService {
   }
 
   getFkMapping(tableName: string, data: IConv): IFkTabData[] {
-    let srcTableName: string = data.ToSource[tableName].Name
-    let spFks = data.SpSchema[tableName].Fks
-    if (!spFks) {
+    //Todo : Need to differentiate between src and spanner table name in argument
+    // let srcTableName: string = data.ToSource[tableName].Name
+    let srcTableName = tableName
+    let spFks =
```

Comment:
Why are we using table name as same value for both source and spanner database ? It may be different, consider the case of having table name as `x@1` in case of MySQL , it would correspond to spanner table as `x_1`

Reply:
Resolve by passing tableId as argument and extracting both sourcename and spannername from id.